test/Interpreter/variadic_generic_opaque_type.swift

```diff
@@ -10,6 +10,9 @@
 
 // REQUIRES: executable_test
 
+// This test needs a Swift 5.9 runtime or newer.
+// UNSUPPORTED: back_deployment_runtime
```

Comment:
@tshortli Eventually, we should make this more fine-grained so that we don't have an ever-growing pile of tests disabled on backward deployment bots.